tests/test_routes.py

```diff
@@ -0,0 +1,153 @@
+from app.routes import validate_model
+from app.models.planet import Planet
+from werkzeug.exceptions import HTTPException
+import pytest
+
+
+
+def test_read_all_planets_with_no_records(client):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_read_one_planet(client, two_saved_planets):
+
+    response = client.get("/planets/2")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == {
+        "id": 2,
+        "name": "Earth",
+        "description": "a little polluted",
+        "color": "purple"
+    }
+def test_create_planet(client):
+    # Arrange
+    new_planet = {
+        "name": "Colorful Planet",
+        "description": "The Best!",
+        "color": "rainbow"
+    }
+    # Act
+    response = client.post("/planets", json=new_planet)
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 201
+    assert response_body == "Planet Colorful Planet successfully created"
```

Comment:
It would be a good idea to follow this up with a get call to confirm that the planet actually got stored in the database.